### PR TITLE
finding misconfigurations in your fabric

### PR DESCRIPTION
**Pull request by cloud-native-everything**
Hi Roman, this is a new post. Let me know if this works. Thanks.

### CODE REVIEW

**Comment by cloud-native-everything** [comment]
Roman, the image is not showing in the deployment. Do you have those disabled?

File: docs/blog/posts/2023/nanog88-python-pygnmi.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+<small>:material-github: [Git Repo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88)</small>
+
+Today, I'm sharing another piece of my experience from the [NANOG88](https://www.nanog.org/events/nanog-88/) conference where I had the privilege of presenting a tutorial featuring pyGNMI, a powerful tool for diagnosing network issues. During my talk, I used pyGNMI to visualize EVPN Layer2 and Layer 3 domains, sorting them by switch or network instance. I also added a special feature that detects discrepancies in the settings between different switches using the same EVPN domain – a great way to catch typos in your BGP/VXLAN settings.
+
+For this demonstration, I leveraged containerlab and SR Linux to build a VXLAN-EVPN Fabric, replicating a typical configuration I often use in my Kubernetes labs. I incorporated eBGP for underlay communication, and the topology I utilized comprised two spines, two leaf switches, and a border leaf.
+
+Check out the full details in the corresponding [Git Repo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88)
+
+## How this app works
```
Comment (hellt) [inline docs/blog/posts/2023/nanog88-python-pygnmi.md:27]: @cloud-native-everything can you also add a short section here on how to deploy the lab if users want to follow your examples?

File: docs/blog/posts/2023/nanog88-python-pygnmi.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+| clab-dc-k8s-LEAF-DC-2 |      l3evpn      | 1  |      enable      |    vxlan1.2     |  2   |  4   |     up     |  1.1.1.2:2   |  target:65123:2   |  target:65123:2   |
++-----------------------+------------------+----+------------------+-----------------+------+------+------------+--------------+-------------------+-------------------+
+Total time: 1.42 seconds
+```
+Now, if you have a typo in the EVI number, then, I added a function will show you that:
+![EVPN configuration typo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88/blob/main/py-scripts/images/Highligthed-Typo-EVPN-Fabric-Configuration.png)
+
+## How to use it
+To use the python class, you'll need to install some modules, including tabulate and pygnmi. Following this, you can import the class as such:
+```python
```
Comment (hellt) [inline docs/blog/posts/2023/nanog88-python-pygnmi.md]: if your script relies on external packages, you have to create `requirements.txt` listing those.

File: docs/blog/posts/2023/nanog88-python-pygnmi.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+from srl_evpn_class import SrlDevice
+from srl_evpn_class import MergeEvpnToArray
+from srl_evpn_class import HighlightAlternateGroups
```
Comment (hellt) [inline docs/blog/posts/2023/nanog88-python-pygnmi.md:74]: this doesn't look like idiomatic python code. You generally want to call the class just as `Srlinux` or `SrlEvpn`.

File: docs/blog/posts/2023/nanog88-python-pygnmi.md (hunk 1 above)
Comment (cloud-native-everything) [reply docs/blog/posts/2023/nanog88-python-pygnmi.md:43]: done

File: docs/blog/posts/2023/nanog88-python-pygnmi.md (hunk 2 above)
Comment (cloud-native-everything) [reply docs/blog/posts/2023/nanog88-python-pygnmi.md]: done

File: docs/blog/posts/2023/nanog88-python-pygnmi.md (hunk 3 above)
Comment (cloud-native-everything) [reply docs/blog/posts/2023/nanog88-python-pygnmi.md]: done

**Comment by cloud-native-everything** [comment]
All changes done, let me know if we are good know. Thanks.

**Review by hellt — APPROVED** [review]

**Pull request merged**